### ISSUE TITLE
Add SR-only description to RADAR component

### ISSUE THREAD

**Issue by colinmurphy01** [weather-gov/weather.gov#1062]
Coming out of our 4/16 manual accessibility review, the following was suggested for improvement
- Add a SR-only description to the radar component, using the following text as a solution
  - “Listed below is a visual radar representation of radar data for [Red Lodge, MT], for more details on specific weather conditions view hourly details within daily tab” (hyperlink to tab)

Heading updates will be made in #1036

**Comment by colinmurphy01** [comment]
@igorkorenfeld @kmranjo thoughts on this?

**Comment by kmranjo** [comment]
We have the first bullet sort of addressed in #1003 although I'm not sure how it works to have it serve as both a heading for the content and a "skip to SR only text" link. 

For the radar, we do also already have a heading noted there. I agree with adding the text you have above, but it would be added as SR text, not a heading.

**Comment by partly-igor** [comment]
Agree with @kmranjo on both.  

For the heading, I think it would mainly serve as a navigation landmark for the screen reader so that if someone navigates by headings they can go to "current conditions" and the next item arrowed to after navigating to the `h2` would be the current condition text. 

And yes I imagine the description would be SR-only text below the "Radar" heading.

**Comment by colinmurphy01** [comment]
So this ticket should be rescoped to just add the SR-only text for radar since the other two are solved by #1036?
